roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -16,7 +16,11 @@
   include: install_heapster.yaml
   when: openshift_metrics_heapster_standalone | bool
 
-- find: paths={{ mktemp.stdout }}/templates patterns=*.yaml
+- name: Install Hawkular OpenShift Agent (HOSA)
+  include: install_hosa.yaml
+  when: openshift_metrics_install_hawkular_agent | bool
+
+- find: paths={{ mktemp.stdout }}/templates patterns="^(?!metrics-hawkular-openshift-agent).*.yaml" use_regex=true
```

Comment:
right, sorry, that's what I'm asking -- if you could split this up to be on separate lines for each param